### ISSUE TITLE
In the Russian version of the plug-in does not display the card

### ISSUE THREAD

**Issue by Soomei** [flowdee/hearthstone-cards-for-wordpress#33]
In the Russian version of the plug-in does not display the card.

Here is a list of some of them:
[Пленный йормунгар]
[Боец арены]
[Ледяной яростень]
[Грозный демон]
[Просоленный моряк]
[Танк-паук]
[Грязевик]
[Речной кроколиск]
[Правосудие Света]
[Огр Тяжелого Кулака]
[Огненная секира]
[Ящер Кровавой Топи]
[Оазисный хрустогрыз]
[Мурлок-налетчик]
[Магмовый яростень]
[Огонек]
[Морозный йети]
[Арканитовый жнец]
[Боевой голем]
[Гончая Недр]
[Клинок убийцы]

Можете самостоятельно проверить это здесь в списке:
http://mulligan.ru/o-nas/

**Comment by Soomei** [comment]
It is also activated, but no images are displayed on cards:
[Боевой настрой]
[Королевский поборник]
[Лавовый шок]
[Улучшение]
[Внутренняя ярость]

And many others...

Screenshot: http://prntscr.com/9jgzj6

**Comment by flowdee** [comment]
Hey Soomei,

The plugin combines the card database of https://hearthstonejson.com/ with the card images from http://ru.hearthhead.com/. In case the card names are written exactly like in the json files AND hearthead hosts the image under the exact same name, it will match.

Anyway there can be some issues with special characters of your language and the wordpress editor/output.

Summing up I cannot read/speak Russian and I'm personally not able to debug this in your case. Sorry.

**Comment by Soomei** [comment]
If you fix the problem, I will tell you

**Comment by flowdee** [comment]
:+1:

**Comment by Soomei** [comment]
You plug-in a reference image : http://wow.zamimg.com/images/hearthstone/cards/ruru/medium/CS2_004e.png

A website http://ru.hearthhead.com/ : http://wow.zamimg.com/images/hearthstone/cards/ruru/original/CS2_004.png?10833

http://ru.hearthhead.com/card=613&text

The problem is probably in this, how to fix it?

**Comment by flowdee** [comment]
Oh yeah thanks for the tip. Seems to be the fact that some cards are _not yet_ available in medium size on hearthhead.

Is this the only problem for the others examples too? Then I'm going think about a workaround for this.

**Comment by Soomei** [comment]
No, these cards are many. Also, some cards are not activated in brackets []
If you need a list, I gather in my spare time.

**Comment by flowdee** [comment]
It would help if you find a pattern for those lacking cards. I can imagine that it's a problem when the script tries to recognize the characters of your language.

**Comment by Soomei** [comment]
where to place the list of ID cards that do not work? task # 33 or # 36 ?
In what form they need to provide?

**Comment by flowdee** [comment]
just post them here
